deploy/crds/crd-challenges.yaml

```diff
@@ -416,6 +416,9 @@ spec:
                                       type: object
                                       additionalProperties:
                                         type: string
+                                overrideNginxIngressWhitelistAnnotation:
+                                  description: OverrideNginxIngressWhitelistAnnotation add description here
```

Comment:
You might need to run

```sh
./hack/update-crds.sh
```

in order to get the descriptions in `crd-challenges.yaml` to get updated.